src/Redux/ConfigureStore.js

```diff
@@ -8,6 +8,6 @@ const reducers = combineReducers({
   categoryReducer,
 });
 
-const store = createStore(reducers);
+const store = configureStore(reducers);
```

Comment:
- [ ] good job working on the previous reviewer suggestions however to use configure store you can remove the combineReducers and do something like this 
```
const store = configureStore({
  reducer: {
    bookReducer: bookReducer,
    categoryReducer: categoryReducer 
  }
});
```